R/descriptives.R

```diff
@@ -1181,6 +1202,64 @@ Descriptives <- function(jaspResults, dataset, options) {
   return(p)
 }
 
+.descriptivesDotPlots <- function(dataset, options, variable){
+  
+  
+  if (options$splitby != "" ) {
+    # return a collection
+    levelsSplitFactor <- names(dataset)
+    
+    plotContainer <- createJaspContainer(title = variable)
+    plotContainer$dependOn(optionContainsValue = list(variables = variable))
+    
+    for (level in levelsSplitFactor)
+      plotContainer[[level]] <- .descriptivesDotPlots_SubFunc(dataset = dataset[[level]], variable = variable, title = level)
+    
+    return(plotContainer)
+    
+  }else{
+    
+    dotplot <- .descriptivesDotPlots_SubFunc(dataset = dataset, variable = variable, title = variable)
+    dotplot$dependOn(optionContainsValue = list(variables = variable))
+    
+    return(dotplot)
+  }
+}
+
+.descriptivesDotPlots_SubFunc <- function(dataset, variable, title){
+  
+  dotplot <- createJaspPlot(title = title)
+  
+  x <- na.omit(dataset[[.v(variable)]])
+  
+  if (is.factor(x)){
+    tb <- as.data.frame(table(x))
+    p <- ggplot2::ggplot(data = data.frame(x = x), ggplot2::aes(x = x)) + 
+      ggplot2::geom_dotplot(binaxis = 'x', stackdir = 'up') + 
+      ggplot2::xlab(variable) +
+      ggplot2::ylab("") +
```

Comment:
```suggestion
      ggplot2::ylab(NULL) +
```
otherwise the plot literally puts `""` on the axis. With `NULL` the space for the axis title is removed completely (and the plot area will be slightly bigger).